game_eggs/steamcmd_servers/7_days_to_die/egg-7-days-to-die.json

```diff
@@ -109,6 +109,16 @@
             "user_editable": true,
             "rules": "required|string|max:20",
             "field_type": "text"
+        },
+        {
+            "name": "Network Protocols",
+            "description": "Networking protocols that should NOT be used. Separated by comma. Possible values: LiteNetLib, SteamNetworking. Dedicated servers should disable SteamNetworking if there is no NAT router in between your users and the server or when port-forwarding is set up correctly. lets it empty if you are connecting your self hosted server behind a NAT",
+            "env_variable": "SERVER_DISABLED_NETWORK_PROTOCOLS",
+            "default_value": "SteamNetworking",
```

Comment:
Please make this default value nothing and make it nullable. The host will have to set it.
Check if this parameter is empty and the flag added to the startup that the server still starts